Assets/Scripts/StaminaBarSetColour.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class StaminaBarSetColour : MonoBehaviour {
+    public SpriteRenderer spriteRenderer;
```

Comment:
why would you use a line renderer to display a wheel, and just so you are aware, my ability to use unity is limited. I felt that this is the best approch considering, there are other approches I know but this is what I know.